src/components/cell-tooltip/CellTooltip.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import expect from 'expect';
+import CellTooltip from './CellTooltip';
+
+configure({ adapter: new Adapter() });
+
+const fakeHoveredCellInfo = {
+  cellId: '123',
+  mappings: { xy: [1, 1], tsne: [2, 2] },
+  uuid: 1,
+  status: 'subcluster: Hippocampus; cluster: Excitatory neurons',
+};
+
+const makeFakeViewInfo = (x, y) => ({
+  width: 10,
+  height: 10,
+  viewport: {
+    project() {
+      return [x, y];
+    },
+  },
+});
+
+const fakeViewInfoInside = makeFakeViewInfo(5, 5);
+const fakeViewInfoOutsideAbove = makeFakeViewInfo(0, -1);
+const fakeViewInfoOutsideBelow = makeFakeViewInfo(0, 11);
+const fakeViewInfoOutsideLeft = makeFakeViewInfo(-1, 0);
+const fakeViewInfoOutsideRight = makeFakeViewInfo(11, 0);
```

Comment:
I would just put a call to `makeFakeViewInfo` inside each test... Try to keep variables on the lowest level that works, and keep the toplevel clean.